File: TestArdunioService/testArduinoService.py
#!/usr/bin/env python
import pika

connection = pika.BlockingConnection(pika.ConnectionParameters(
        host='localhost'))
channel = connection.channel()

channel.queue_declare(queue='GreenhouseData')

channel.basic_publish(exchange='',
                      routing_key='GreenhouseData',
                      body='GreenhouseTalking')
print " [x] Sent 'GreenhouseTalking'"
connection.close()

File: services/arduino_test_Service/testArduinoService.rb
#!/usr/bin/env ruby
# encoding: utf-8

require "bunny"

conn = Bunny.new(:automatically_recover => false)
conn.start

ch   = conn.create_channel
q    = ch.queue("GreenhouseData")

ch.default_exchange.publish("greenhouseTalking", :routing_key => q.name)
puts " [x] Sent GreenhouseTalking!"

sleep 0.5
conn.close